src/esm_parser/esm_parser.py

```diff
@@ -478,15 +478,28 @@ def attach_to_config_and_reduce_keyword(
                 else:
                     tmp_config = include_path
 
-                config_to_write_to[tmp_config["model"]] = tmp_config
+                # TODO: remove this if and always do merging after component keys in all yamls
+                # old ways...
+                if "model" in tmp_config:
+                    config_to_write_to[tmp_config["model"]] = tmp_config
+                # new way!
+                else:
+                    # TODO: add an if if computer changes are to be ignored for the standalone files
+                    dict_merge(config_to_write_to, tmp_config)
```

Comment:
Where would the switch for this `if` be? Hopefully not in `dict_merge`, but in the function signature?

Reply:
This is already handled somewhere else. The switch is handled in the esm_environment.py with a var called `include_env_from_component_files`, which is by default set to true